files/nrpe/check_openstack.py

```diff
@@ -322,9 +342,9 @@ def delete_orphaned_instances(self):
       instance.delete()
 
   def delete_orphaned_floating_ips(self):
-    for tenant_ip in self.nova.floating_ips.list():
-      self.nova.floating_ips.delete(tenant_ip)
-    if len(self.nova.floating_ips.list()) != 0:
+    for tenant_ip in self.neutron.list_floatingips()['floatingips']:
+      self.neutron.delete_floatingip(tenant_ip['id'])
```

Comment:
From video review with team: This should be checked that it doesn't do anything dangerous if run as admin.

Reply:
Confirmed, when creating a network for nagiostest and running the test as nagiostest, the check removed IP from an unrelated canary. This should get a safeguard, even if right now we don't run it as admin user.

Reply:
Added a minimum viable safeguard.